### ISSUE TITLE
Official upload to the Firefox Addon Store

### ISSUE THREAD

**Issue by APT37** [go-shiori/shiori-web-ext#72]
An official upload (via CI/CD?) would make any 3rd party uploads obsolete and also prevent problems like [this](https://github.com/go-shiori/shiori-web-ext/issues/70) from happening.

There are two versions available on AMO, *both outdated*:

- [Shiori](https://addons.mozilla.org/en-US/firefox/addon/shiori_ext/) (v0.8.5, occupying the original UUID)
- [Shiori extension](https://addons.mozilla.org/en-US/firefox/addon/shiori-extension/) (v1.3)

I refrained from creating another issue along the lines of 'Current version unavailable to Android users', because the solution is ultimately the same. ([XPI sideloading is a hidden feature in Firefox for Android.](https://github.com/go-shiori/shiori-web-ext/issues/70#issuecomment-2271566094))

**If you have problems logging into the Shiori extension, please see this comment:** https://github.com/go-shiori/shiori-web-ext/issues/70#issuecomment-2101821474

**Comment by yonas** [comment]
What's the objection for uploading to AMO?

**Comment by fmartingr** [comment]
It is possible to automate this on the CI on release?

**Comment by edsu** [comment]
I'm just chiming in here to say I was *very* confused by the out of date Shiori extension on Firefox Addon Store. I thought something was wrong with my Shiori configuration! After seeing this issue I tried to build the extension myself and run it temporarily in my Firefox browser and it worked.  Then I had to get an AMO account to sign it myself so I could install it permanently. Only after that did I read the fine print and realize the XPI file was here in the releases. D'oh! A lot of work for nothing :-) But thank you for the extension, and for shiori!

**Comment by fmartingr** [comment]
Anyone knows if is there a process to reclaim the addon UUID? I checked and the current build process is uploading it as unlisted over a new UUID. I just made the migration to manifest v3 and wanted to test out publishing it to the listed channel.

**Comment by fmartingr** [comment]
I just sent him an email. Will keep you updated.

On the worst case, I just need to reupload the plugin with a different UUID, which is not that big of a deal, but extension users not aware of this conversation won't be able to update.

**Comment by fmartingr** [comment]
Nover quickly answered so I'm happy to announce that the latest shiori extension version is now available in the firefox store: https://addons.mozilla.org/en-US/firefox/addon/shiori_ext/

Thanks Nover for your help with this!
